x-pack/plugins/asset_inventory/server/create_transforms/create_transforms.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { transformError } from '@kbn/securitysolution-es-utils';
+import { TransformPutTransformRequest } from '@elastic/elasticsearch/lib/api/types';
+import type { ElasticsearchClient, Logger } from '@kbn/core/server';
+import { errors } from '@elastic/elasticsearch';
+
+// TODO: Move transforms to integration package
+export const initializeTransforms = async (
+  esClient: ElasticsearchClient,
+  logger: Logger
+): Promise<void> => {
+  // Deletes old assets from previous versions as part of upgrade process
+  await deletePreviousTransformsVersions(esClient, logger);
+  // TODO initialize transforms here
+  // await initializeTransform(esClient, <TRANSFORM_HERE>, logger);
```

Comment:
Placeholder for transforms initialization. Please let me know if this is enough or if I should uncomment the line and add some specific transforms